packages/payment-detection/test/any/any-to-eth.test.ts

```diff
@@ -68,7 +68,8 @@ describe('Any to ETH payment detection', () => {
     },
   };
 
-  it('RPC Payment detection', async () => {
+  // FIXME migrate to goerli or mock RPC call
```

Comment:
RPC call is already mocked here
```suggestion
  // FIXME migrate to goerli
```